scripts/run_locally_with_docker.sh

```diff
@@ -21,10 +20,9 @@ else
   EXPOSED_PORTS=""
 fi
 
-docker run -it --rm \
+docker run -it -d --name notifications_api --rm \
```

Comment:
I'm not sure this `-d` flag is what most people would expect ... suggests we're doing something a bit unusual here.

Reply:
Yep, I'd expect this to run in the foreground I think

Reply:
ok